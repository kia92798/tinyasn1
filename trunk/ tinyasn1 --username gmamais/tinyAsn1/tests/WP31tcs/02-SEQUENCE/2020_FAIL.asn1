--FSemantix Error: COMPONENTS OF  should point to a non built in type
--EMY-MODULE DEFINITIONS ::= BEGIN
--E&nbsp;&nbsp;OneType ::= SEQUENCE {
--E&nbsp;&nbsp;&nbsp;&nbsp;a&nbsp;&nbsp;IA5String,
--E&nbsp;&nbsp;&nbsp;&nbsp;COMPONENTS OF INTEGER,&nbsp;&nbsp;
--E&nbsp;&nbsp;&nbsp;&nbsp;b&nbsp;&nbsp;UniversalString
--E&nbsp;&nbsp;}
--EEND

MY-MODULE DEFINITIONS ::= BEGIN
	
	OneType ::= SEQUENCE {
		a	IA5String,
		COMPONENTS OF INTEGER,	-- Error is here
		b	UniversalString
	}
		
END
