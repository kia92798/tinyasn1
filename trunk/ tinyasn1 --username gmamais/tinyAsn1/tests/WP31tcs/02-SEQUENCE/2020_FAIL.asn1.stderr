Error line: 14. INTEGER is not SEQUENCE or SET
